### PR TITLE
Share to meeting in stage view sample

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Review by Pawan-MSFT — APPROVED** [review]
looks correct, approving.

**Review by Pawan-MSFT — APPROVED** [review]
looks correct, Approving.

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/share-to-meeting.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                class="teams-share-in-meeting-button"
+                data-href="<Application-Base-URL>"
+                data-app-id="<Application-ID>"
+                data-entity-name="meeting-test-app"
+				data-button-type="medium"
+				data-icon-px-size="160"
+                data-entity-description="meeting test page to showcase the use of share to meeting functionality"
+            >
+            </div>
+		</div>
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/components/share-to-meeting.jsx:33]: Code alignment doesn't looks correct.

File: samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/router/router.js [hunk 2]
```diff
@@ -28,6 +29,7 @@ export const AppRoute = () => {
         <React.Fragment>
             <BrowserRouter>
                 <Routes>
+		    <Route path="/" element={<ShareToMeeting />}/>
                     <Route path="/appInMeeting" element={<AppInMeeting />}/>
                     <Route path="/configure" element={<Configure />}/>
                     <Route path="/home" element={<Home />}/>
```
Comment (ChetanSharma-msft) [inline samples/meetings-stage-view/csharp/AppInMeeting/ClientApp/src/router/router.js:35]: Looks like Spacing is not correct, please check once ?
{element={<ShareToMeeting />}   

element= { <ShareToMeeting /> }

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3779646685
Status:

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline again: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3788668817

**Pull request merged**